sharktank/sharktank/layers/ffn_block.py

```diff
@@ -22,18 +23,29 @@ class FFN(ThetaLayer):
     def __init__(
         self,
         theta: Theta,
+        is_gated: bool = None,
```

Comment:
How about setting a default boolean value here instead of None.

Reply:
Having the default as None helps with propagation of the underlying default value to callers. They can easily say that their default should change with the default of the underlying function.
E.g.
```
def f(x: bool = None)
    ...

def g(x: bool = None):
    f(x)
```

Reply:
I changed the default to `True` to avoid confusion.